spec/ruby/core/gc/stat_spec.rb

```diff
@@ -13,4 +13,16 @@
     stat.should be_kind_of(Hash)
     stat.keys.should include(:count)
   end
+
+  it "increases count after GC is run" do
+    count = GC.stat(:count)
+    GC.start
+    (GC.stat(:count) > count).should be_true
```

Comment:
We should use `GC.stat(:count).should > count` here to get a better error (it'll show the values) if it fails. I'll do that.